[WIP] Documentation for Logging 5.1 Release Notes (#32927)
openshift/openshift-docs: logging/cluster-logging-release-notes.adoc

[id="cluster-logging-release-notes"]

> = Release notes for Red Hat OpenShift Logging 5.1

Reviewer: ```suggestion
= Release notes for Red Hat OpenShift Logging 5.1 afdkaslfkjdsafjlk
```

include::modules/cluster-logging-document-attributes.adoc[]
:context: cluster-logging-release-notes-v5x

toc::[]

[id="openshift-logging-5-1-0-about-this-release"]
== About this release

> The following advisories are available for {ProductName} 5.1:

Author: ```suggestion
The following advisories are available for {ProductName} 5.1: asjdlaksjd;asdja;sd
```

* link:https://access.redhat.com/errata/RHSA-2021:2112[RHSA-2021:2112 OpenShift Logging Bug Fix Release 5.1.0]

> * link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 Security Advisory for Important OpenShift Logging Bug Fix Release (5.0.3)]

Reviewer: Remove

[id="openshift-logging-5-1-0-inclusive-language"]
== Making open source more inclusive

Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].

[id="openshift-logging-5-1-0-deprecated-removed-features"]
== Deprecated and removed features

Some features available in previous releases have been deprecated or removed.

Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.

[id="openshift-logging-5-1-0-elasticsearch-curator"]

> === Elasticsearch Curator

Reviewer: ```suggestion
=== Elasticsearch Curator has been removed
```

Author: Done.

> The Elasticsearch Curator, which was deprecated previously, in OpenShift Logging 5.0 and is removed in OpenShift Logging 5.1. Elasticsearch Curator helped you curate or manage your indices on OpenShift Container Platform 4.4 and earlier. Now, instead of using Elasticsearch Curator,  xref:../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[configure the log retention time].

Reviewer: ```suggestion
The Elasticsearch Curator has been removed and is no longer supported. Elasticsearch Curator helped you curate or manage your indices on OpenShift Container Platform 4.4 and earlier. Now, instead of using Elasticsearch Curator,  xref:../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[configure the log retention time].
```

Reviewer: This rewording should bring this removal notice in-line with other OpenShift removal notices. Sorry for the late change.

Author: Thanks! Done.

// Release Notes by version
include::modules/cluster-logging-release-notes-5.1.0.adoc[leveloffset=+1]

> include::modules/cluster-logging-release-notes-5.0.3.adoc[leveloffset=+1]

Reviewer: Remove this line.

[id="cluster-logging-release-notes"]
= Release notes for Red Hat OpenShift Logging 5.1
include::modules/cluster-logging-document-attributes.adoc[]
:context: cluster-logging-release-notes-v5x

toc::[]

[id="openshift-logging-5-1-0-about-this-release"]
== About this release

The following advisories are available for {ProductName} 5.1:

* link:https://access.redhat.com/errata/RHSA-2021:2112[RHSA-2021:2112 OpenShift Logging Bug Fix Release 5.1.0]
* link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 Security Advisory for Important OpenShift Logging Bug Fix Release (5.0.3)]
* link:https://access.redhat.com/errata/RHBA-2021:1167[RHBA-2021:1167 Bug Fix Advisory for OpenShift Logging Bug Fix Release (5.0.2)]
* link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 Bug Fix Advisory for OpenShift Logging Bug Fix Release (5.0.1)]

> * link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 Bug Fix Advisory for OpenShift Logging 5.0.0]

Reviewer: We can remove these 5.x release notes.

Author: Done.

[id="openshift-logging-5-1-0-inclusive-language"]
== Making open source more inclusive

Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].

> [id="openshift-logging-5-1-0-deprecated-removed-features"]

Reviewer: ```suggestion
[id="openshift-logging-5-1-new-features-and-enhancements"]
== New features and enhancements

This release adds improvements related to the following components and concepts.

* <insert enhancement release note here> 

[id="openshift-logging-5-1-0-deprecated-removed-features"]
```

== Deprecated and removed features

Some features available in previous releases have been deprecated or removed.

Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.

[id="openshift-logging-5-1-0-elasticsearch-curator"]
=== Elasticsearch Curator

> The Elasticsearch Curator is deprecated in OpenShift Logging 5.0 and is removed in OpenShift Logging 5.1. Elasticsearch Curator helps you curate or manage your indices on OpenShift Container Platform 4.4 and earlier. Instead of using Elasticsearch Curator,  xref:../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[configure the log retention time].

Reviewer: ```suggestion
The Elasticsearch Curator, which was deprecated previously, is removed in OpenShift Logging 5.1. Elasticsearch Curator helped you curate or manage your indices on OpenShift Container Platform 4.4 and earlier. Now, instead of using Elasticsearch Curator,  xref:../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[configure the log retention time].
```

Author: Done.

// Release Notes by version
include::modules/cluster-logging-release-notes-5.1.0.adoc[leveloffset=+1]
include::modules/cluster-logging-release-notes-5.0.3.adoc[leveloffset=+1]
include::modules/cluster-logging-release-notes-5.0.2.adoc[leveloffset=+1]
include::modules/cluster-logging-release-notes-5.0.1.adoc[leveloffset=+1]

> include::modules/cluster-logging-release-notes-5.0.0.adoc[leveloffset=+1]

Reviewer: Remove

> include::modules/cluster-logging-release-notes-5.0.0.adoc[leveloffset=+1]

Reviewer: Remove

Author: Done.

[id="cluster-logging-release-notes"]

> = Release notes for Red Hat OpenShift Logging 5.1

Author: Sure, Done. PTAL.

include::modules/cluster-logging-document-attributes.adoc[]
:context: cluster-logging-release-notes-v5x

toc::[]

[id="openshift-logging-5-1-0-about-this-release"]
== About this release

> The following advisories are available for {ProductName} 5.1:

Author: Sure, Done. PTAL.

* link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 Bug Fix Advisory for OpenShift Logging Bug Fix Release (5.0.1)]
* link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 Bug Fix Advisory for OpenShift Logging 5.0.0]

[id="openshift-logging-5-1-0-inclusive-language"]
== Making open source more inclusive

Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].

[id="openshift-logging-5-1-0-deprecated-removed-features"]
== Deprecated and removed features

Some features available in previous releases have been deprecated or removed.

Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.

> [id="openshift-logging-5-1-0-elasticsearch-curator"]

Reviewer: Please move this 5.1-specific information (lines 25-35) to modules/cluster-logging-release-notes-5.1.0.adoc

Author: Done. PTAL and let me know if all is good.

Reviewer: /lgtm

=== Elasticsearch Curator has been removed

> The Elasticsearch Curator has been removed and is no longer supported. Elasticsearch Curator helped you curate or manage your indices on OpenShift Container Platform 4.4 and earlier. Now, instead of using Elasticsearch Curator,  xref:../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[configure the log retention time].

Author: Done.
